controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java

```diff
@@ -147,6 +147,14 @@ public Lock lockProvisionState(String provisionStateId) {
         return lock(lockPath(provisionStateId), Duration.ofMinutes(30));
     }
 
+    public Lock lockVespaServerPool() {
+        return lock(root.append("locks").append("vespaServerPoolLock"), defaultLockTimeout);
```

Comment:
"lock" is already in the path so no need for it in the node name imho

Reply:
Also, updating the server pool should never take more than a few seconds at most, as it's always just writing data to ZK - so we could probably use a much shorter lock timeout

Reply:
Guess it makes sense to have short timeout, but I'll keep path the same so at least its consistent with the already existing locks.